canonical-web-and-design/cookie-policy
# feat: Automatically trigger initialisation if cookiePolicy script is ran after DOM content has loaded

## Done

- Adds a fallback to automatically trigger the `init` function if the `cookiePolicy` script is called after DOM content has loaded. I also as added a check with in the `init` function to see if it has already ran to avoid it being ran twice.

## QA

- Check out this branch
- Build the project with `npm run build`
- Run the project with `npm run server`
- Visit the website at http://0.0.0.0:8301/
- Revoke the cookies
- Check that the cookie-policy modal opens
- Defer the cookie-policy script from running until after DOM content has loaded (you can do this by adding the `defer` attribute to the script tag)
